README.md

```diff
@@ -147,46 +147,131 @@ Graphjet includes [CLICK, FAVORITE, RETWEET, REPLY, AND TWEET as input node type
 
 [Input features](https://github.com/twitter/the-algorithm-ml/blob/main/projects/home/recap/FEATURES.md) 
 
-Outputs are predictions on how user will respond to the tweet: 
-+ probability the user will favorite the Tweet
-+ probability the user will click into the conversation of this tweet and reply or like a Tweet
-+ probability the user will click into the conversation of this Tweet and stay there for at least 2 minutes.
-+probability the user will react negatively (requesting "show less often" on the Tweet or author, block or mute the Tweet author) 
-+ probability the user opens the Tweet author profile and Likes or replies to a Tweet
-+ probability the user replies to the Tweet
-+ probability the user replies to the Tweet and this reply is engaged by the Tweet author 
-+ probability the user will click Report Tweet 
-+ probability the user will ReTweet the Tweet
-+ probability (for a video Tweet) that the user will watch at least half of the video
-
 All of these are combined and weighted into a score. Hyperparameters for the model [and weighting are here.](https://github.com/twitter/the-algorithm-ml/blob/78c3235eee5b4e111ccacb7d48e80eca019e480c/projects/home/recap/config/local_prod.yaml#L1)
 
 For more details on the model, see the [Architecture overview](masknet.md).
 
-+ The Light Ranker
+
+### Scoring Plan
+
+After the model predicts the probability of the actions, weights are assigned to the probability. The tweet with the highest score is likely to appear at the top of your feed.
+
+These are the actions predicted, and [their corresponding weights](https://raw.githubusercontent.com/twitter/the-algorithm-ml/main/projects/home/recap/README.md)
++ probability the user will favorite the Tweet (0.5)
++ probability the user will click into the conversation of this tweet and reply or like a Tweet (11*)
++ probability the user will click into the conversation of this Tweet and stay there for at least 2 minutes (11*)
++ probability the user will react negatively (requesting "show less often" on the Tweet or author, block or mute the Tweet author) (-74)
++ probability the user opens the Tweet author profile and Likes or replies to a Tweet (12)
++ probability the user replies to the Tweet (27)
++ probability the user replies to the Tweet and this reply is engaged by the Tweet author (75)
++ probability the user will click Report Tweet (-369)
++ probability the user will ReTweet the Tweet (1)
++ probability (for a video Tweet) that the user will watch at least half of the video (0.005)
+
+The score of the tweet is equal to
+
+P(favorite) * 0.5 + max( P(click and reply), P(click and stay two minutes) ) * 11 + P(hide or block or mute) * -74 + ... etc
+
+The tweet with the highest score is likely to appear at the top of your feed. (There is still a part on boost where multipliers will be applied to the score).
```

Comment:
Maybe add in some notes about how they add filters afterwards to the ranked candidates that could change this?